### PR TITLE
Duplicate contract names (mat-1649, mat-1648)

### PR DESCRIPTION
**Pull request by AyushKaul**
Tests are not passing for slashingManager `Slashing:validator` and `Slashing:delegation`

### CODE REVIEW

File: contracts/staking/StakingInfo.sol [hunk 1]
```diff
@@ -404,8 +404,10 @@ contract StakingInfo {
             _contract,
             status
         ) = stakeManager.validators(validatorId);
-        reward += IStakeManager(_contract).validatorRewards();
-        _status = uint256(status);
+        if (_contract != address(0x0)) {
+            reward += IStakeManagerLocal(_contract).validatorRewards();
+            _status = uint256(status);
```
Comment (0xAshish) [inline contracts/staking/StakingInfo.sol:409]: Status should be outside of this if condition.

Comment (AyushKaul) [reply contracts/staking/StakingInfo.sol]: done

File: test/units/staking/stakeManager/StakeManager.Staking.js [hunk 2]
```diff
@@ -348,10 +348,10 @@ module.exports = function(accounts) {
         })
       })
 
-      // for some reason DummyERC20 doesn't have Transfer event
+      // for some reason TestToken doesn't have Transfer event
```
Comment (0xAshish) [inline test/units/staking/stakeManager/StakeManager.Staking.js:351]: please remove this comment.

File: test/units/staking/stakeManager/StakeManager.Staking.js [hunk 3]
```diff
@@ -348,10 +348,10 @@ module.exports = function(accounts) {
...
 
       // it('must emit Transfer', async function() {
-      //   await expectEvent.inTransaction(this.receipt.tx, DummyERC20, 'Transfer', {
+      //   await expectEvent.inTransaction(this.receipt.tx, TestToken, 'Transfer', {
```
Comment (0xAshish) [inline test/units/staking/stakeManager/StakeManager.Staking.js:354]: uncomments these tests.

File: test/units/staking/stakeManager/StakeManager.Staking.js [hunk 4]
```diff
@@ -409,10 +409,10 @@ module.exports = function(accounts) {
...
```
Comment (0xAshish) [inline test/units/staking/stakeManager/StakeManager.Staking.js:415]: This one as well.

File: test/units/staking/stakeManager/StakeManager.Staking.js [hunk 5]
```diff
@@ -409,10 +409,10 @@ module.exports = function(accounts) {
         })
       })
 
-      // for some reason DummyERC20 doesn't have Transfer event
+      // for some reason TestToken doesn't have Transfer event
```
Comment (0xAshish) [inline test/units/staking/stakeManager/StakeManager.Staking.js:412]: please remove this as well.

**Pull request merged**